### PR TITLE
Component/header

### PR DESCRIPTION
**Pull request by Tim-Pet**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -8,16 +8,42 @@
     <link rel="stylesheet" href="styles/styles.css" />
   </head>
   <body>
-    <header class="app-header">
-      <h1 class="app-header__heading">Quiz App</h1>
+    <header class="header">
+      <div class="flex mx-4 my-1 flex-align-center">
```
Comment (macfrei) [inline src/index.html:12]: Can you not give these styles the `header` element directly?

File: src/index.html [hunk 2]
```diff
@@ -8,16 +8,42 @@
...
+        <img class="header__profile" src="../../assets/images/profile-icon.png" alt="" />
+        <h1>Hello, <strong>{User}</strong></h1>
+        <span class="header__dots">...</span>
+      </div>
     </header>
 
     <section class="quiz-card mx-4 my-4 px-3 py-3">
```
Comment (macfrei) [inline src/index.html:19]: The quiz-cards should be the main content on your website and therefore wrapped in a `main` element.

**Review by macfrei — COMMENTED** [review]
Good job 👍 One thing to consider: If you have another `header` element on your page, I would stick with naming the main header

File: src/index.html (hunk 1 above)
Comment (Tim-Pet) [reply src/index.html:12]: true 🤦🏼

**Pull request merged**